docker-compose.yml

```diff
@@ -22,14 +22,9 @@ services:
       - CACHE_AXES=redis:6379/0
       - DISABLE_2FA=yes
       - SUBPATH=${SUBPATH:-/}
-      # setup_configuration env vars
-      - SITES_CONFIG_ENABLE=yes
-      - OBJECTTYPES_DOMAIN=web:8000
-      - OBJECTTYPES_ORGANIZATION=ObjectTypes
-      - OBJECTS_OBJECTTYPES_CONFIG_ENABLE=yes
-      - OBJECTS_OBJECTTYPES_TOKEN=some-random-string
-      - OBJECTS_OBJECTTYPES_PERSON=Some Person
-      - OBJECTS_OBJECTTYPES_EMAIL=objects@objects.local
+      - RUN_SETUP_CONFIG=${RUN_SETUP_CONFIG:-true}
+    volumes: &web-volumes
+      - ./docker/setup_configuration:/app/setup_configuration
```

Comment:
[This comment](https://github.com/maykinmedia/open-klant/pull/297#discussion_r1869283887) applies here too.

Reply:
Fixed